tests/integration/cmd_project_test.go

```diff
@@ -41,13 +41,12 @@ var _ = Describe("odo project command tests", func() {
 	Context("when running project command app parameter in directory that doesn't contain .odo config directory", func() {
 		It("should successfully execute list along with machine readable output", func() {
 			helper.CmdRunner("odo", "project", "list")
-			time.Sleep(1 * time.Second)
 			listOutput := helper.CmdShouldPass("odo", "project", "list")
 			Expect(listOutput).To(ContainSubstring(project))
 
 			// project deletion doesn't happen immediately, so we test subset of the string
 			listOutputJson := helper.CmdShouldPass("odo", "project", "list", "-o", "json")
-			Expect(listOutputJson).To(ContainSubstring(`{"kind":"Project","apiVersion":"odo.openshift.io/v1alpha1","metadata":{"name":"` + project + `","namespace":"` + project + `","creationTimestamp":null},"spec":{},"status":{"active":false}}`))
+			Expect(listOutputJson).To(ContainSubstring(`{"kind":"Project","apiVersion":"odo.openshift.io/v1alpha1","metadata":{"name":"` + project + `","namespace":"` + project + `","creationTimestamp":null},"spec":{},"status":{"active":true}}`))
```

Comment:
In parallel run validating the projects status ```"status":{"active":true}}``` or ```"status":{"active":true}}``` could lead to a flaky behavior.  Instead of matching the full json can you match some major key value pair.

For now see how this is being done in one of our test cases - https://github.com/openshift/odo/blob/master/tests/integration/component.go#L197